### DISCUSSION TITLE
Bold messages in chat that you are tagged in.

### DISCUSSION THREAD

**Discussion by dooperdoops** [ppy/osu#29074] [Ideas] [answered]
When out of chat, you get notifications when someone tags you and you can click it to get sent to the area of chat it is from. However it doesn't single this message out and especially for users who can't read the smaller text, it's hard to notice where the message tagging you is. I propose a slight light grey highlight around the message box that makes it easier to notice.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/29314
